.github/dependabot.yaml

```
@@ -0,0 +1,9 @@
+# Copyright 2023 SECO Mind Srl
+# SPDX-License-Identifier: Apache-2.0
+
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: weekly
```

Add new line at end of file
```suggestion
      interval: weekly
      
```

---

Done